packages/core/src/controllers/ApiController.ts

```diff
@@ -230,17 +230,16 @@ export const ApiController = {
       ApiController.fetchNetworkImages(),
       ApiController.fetchConnectorImages()
     ]
-    if (OptionsController.state.enableAnalytics === undefined) {
-      promises.push(ApiController.fetchAnalyticsConfig())
-    }
+
     state.prefetchPromise = Promise.race([Promise.allSettled(promises), CoreHelperUtil.wait(3000)])
   },
 
-  async fetchAnalyticsConfig() {
-    const { isAnalyticsEnabled } = await api.get<ApiGetAnalyticsConfigResponse>({
+  async fetchProjectConfig() {
+    const { isAnalyticsEnabled } = await api.get<ApiGetProjectConfigResponse>({
       path: '/getAnalyticsConfig',
       headers: ApiController._getApiHeaders()
     })
-    OptionsController.setEnableAnalytics(isAnalyticsEnabled)
+
+    return { isAnalyticsEnabled, isAppKitAuthEnabled: true }
```

Comment:
If it's already hardcoded as `true`, why do we need to return this value from this function? Will it be read from the response of any endpoint later?

Reply:
Also the project name changed but it still uses `/getAnalyticsConfig`, is it expected?

Reply:
Yes and yes it's temporary until `/getProjectConfig` is live on prod.